test/Main.hs

```diff
@@ -79,7 +79,7 @@ cases
               , expectFail $ runTestForFile "DupFun.solc" caseFolder
               , runTestForFile "DuplicateFun.solc" caseFolder
               , runTestForFile "EitherModule.solc" caseFolder
-              , expectFail $ runTestForFile "Id.solc" caseFolder
+              , runTestForFile "Id.solc" caseFolder
```

Comment:
It's still not completetely working:
```
PANIC: spec(let f : t_lambda_impl01(word, n) = id() ;): free type vars in t_lambda_impl01(word, n)
```

This is due to the fact that main gets desugared to
```
   function main () -> word {
      let f : t_lambda_impl01(word, n) = id() ;
      return invokable.invoke(f, 0);
```

while I think it probably should be `let f : t_lambda_impl01(word, n)`

Reply:
Ah, I see it's probably fixed in #65. Well done :)